Software-Engineering.md

```diff
@@ -145,9 +145,9 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I actively blog about the lessons we are learning as we improve engineering excellence to clarify our thinking, reinforce the learning, attract customers and potential future employees
 - I have a proven track record of expertly tailoring my communication to the target audience with a result of increased clarity and impact
 
-## Engineering Manager
+## Tech Lead / Engineering Manager
```

Comment:
(summary of chat with @hogfish)

This page covering Software Engineer roles will be completely replaced by https://github.com/OctopusDeploy/People/pull/38.

On the old TL/EM role described here: We'll remove this old definition entirely and replace it with the updated, separate roles for technical and management tracks (https://github.com/OctopusDeploy/People/pull/38 and this PR, respectively). Where there is a need for people to simultaneously take on technical and management responsibilities in future (for example, when seeding and growing a new team), Octonauts should generally still identify the _primary track_ that the person is growing along in the longer term, and use that role definition.